content/account_management/index.md

```diff
@@ -0,0 +1,83 @@
+
+**Each invoice is determined by the high watermark of concurrently running hosts for that month.**
+
+This is per active host in Datadog, whether or not it's running the agent.
+
+## Billing FAQ
+
+* [How does the usage details page help me understand my monthly usage?](https://help.datadoghq.com/hc/en-us/articles/115004097483-How-does-the-usage-details-page-help-me-understand-my-monthly-usage-)
+
+* [Where do I find an overview of your pricing?](https://help.datadoghq.com/hc/en-us/articles/214616926-Where-do-I-find-an-overview-of-your-pricing-)
+
+* [Do you support hourly pricing?](https://help.datadoghq.com/hc/en-us/articles/216094043-Do-you-support-hourly-pricing-)
+
+* [How will an AWS integration impact my monthly billing?](https://help.datadoghq.com/hc/en-us/articles/214615086-How-will-an-AWS-integration-impact-my-monthly-billing-)
+
+* [How will a Google Cloud integration impact my monthly billing?](https://help.datadoghq.com/hc/en-us/articles/115000568966-How-will-a-Google-Cloud-integration-impact-my-monthly-billing-)
+
+* [How will a VMware vSphere integration impact my monthly billing?](https://help.datadoghq.com/hc/en-us/articles/115001169326-How-will-a-VMware-vSphere-integration-impact-my-monthly-billing-)
+
+* [How will custom metrics impact my monthly bill?](https://help.datadoghq.com/hc/en-us/articles/115001797986-How-will-custom-metrics-impact-my-monthly-bill-)
+
+* [How do you charge for containers?](https://help.datadoghq.com/hc/en-us/articles/218016463-How-do-you-charge-for-containers-)
+
+* [What plan am I signed up for when I use my credit card?](https://help.datadoghq.com/hc/en-us/articles/214609786-What-plan-am-I-signed-up-for-when-I-use-my-credit-card-)
+
+* [How do I update my credit card?](https://help.datadoghq.com/hc/en-us/articles/115002920306-How-do-I-update-my-credit-card-)
+
+* [How are my bills calculated each month?](https://help.datadoghq.com/hc/en-us/articles/216099263-How-are-my-bills-calculated-each-month-)
+
+* [How do I see what I’ll get charged for this current month?](https://help.datadoghq.com/hc/en-us/articles/216095263-How-do-I-see-what-I-ll-get-charged-for-this-current-month-)
+
+* [Do non-reporting or inactive hosts count towards my bill?](https://help.datadoghq.com/hc/en-us/articles/203037549-Do-non-reporting-or-inactive-hosts-count-towards-my-bill-)
+
+* [What are Kubernetes pause containers and will they impact my bill?](https://help.datadoghq.com/hc/en-us/articles/115002102206-What-are-Kubernetes-pause-containers-and-will-they-impact-my-bill-)
+
+* [How can I change the Billing contact?](https://help.datadoghq.com/hc/en-us/articles/203764785-How-can-I-change-the-Billing-contact-)
+
+* [I don't see anything in my billing history](https://help.datadoghq.com/hc/en-us/articles/203844125-I-don-t-see-anything-in-my-billing-history)
+
```

Comment:
Lets delete this whole page and link to the KB.

Reply:
👍